packages/config/src/load.js

```diff
@@ -66,5 +78,72 @@ export async function loadNuxtConfig ({
     options.rootDir = rootDir
   }
 
+  // Load env to options._env
+  options._env = env
+
+  // Expand and interpolate runtimeConfig from _env
+  for (const c of ['publicRuntimeConfig', 'privateRuntimeConfig']) {
+    if (options[c]) {
+      expand(options[c], options._env)
+    }
+  }
+
   return options
 }
+
+function loadEnv (rootDir) {
+  const env = Object.create(null)
+
+  // Read dotenv
+  const dotenvFile = path.resolve(rootDir, '.env')
+
+  if (fs.existsSync(dotenvFile)) {
+    const parsed = dotenv.parse(fs.readFileSync(dotenvFile, 'utf-8'))
+    Object.assign(env, parsed)
+  }
+
+  // Apply process.env
+  Object.assign(env, process.env)
```

Comment:
Do we really want to always inject the PATH, PWD, __SCRIPT_PID into the env?

If we would pass process.env as second arg to expand a user could choose to inject this by just setting `PATH=` in his/her .env file

Reply:
God point but this way we should either filter process.env or not support real environment variables at all! `.env` is sometimes good for environments that cannot or are not easy to use os environments but not a replacement for it.

Reply:
Also, this is dotenv behavior: https://github.com/motdotla/dotenv#what-happens-to-environment-variables-that-were-already-set

Reply:
e809dbfda93f783dbdb640f0cfe06bad79c24993 can be disabled with envConfig.systemVars

Reply:
nice, thx